app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -47,6 +47,7 @@ def as_json
           return if private_endpoint.blank?
           {
             url: private_endpoint,
+            owner_id: backend_api_id,
```

Comment:
Well I really prefer not using owner_id
We are supposed to be compayible with any other gateway.

The `owner_type` and `owner_id` are internal to Porta
Unless you want them to be part of a specification of Gateway configuration

Reply:
I presented the options to apicast team. They say adding `owner_id` to the policy rules should be enough.

I don't want to make it such big of a deal. For now, moving on with `owner_id` and if they change their mind and prefer something else, not hard to change anyway.

Reply:
Yes I understand it is not a big deal but it is just that as specification it is not really clean and once it is out it is hard to go back. Sure it is just a word but it can make a difference.
Up to you

Reply:
Let's see how it goes: https://github.com/3scale/APIcast/pull/1125#discussion_r334571141

Reply:
I agree with your comment there @guicassolato , imo it should be either `backend_api_id` or `owner_id` + `owner_type`, but only `owner_id` is a bit asking for problems :smile:

Reply:
@guicassolato It would be nice to add `owner_type` and done with this (aka can be merged ;) )

Reply:
It's apicast that needs to add `owner_type`, not us, and they did.

Reply:
Oh. I get it now. For semantics you mean. Only to make it clear who the "owner" is. Ok I will add it.

Reply:
Done.